internal/errors/errors.go

```diff
@@ -0,0 +1,50 @@
+// Package errors contains helper functions for wrapping errors with stack traces, stack output, and panic recovery.
+package errors
+
+import (
+	"fmt"
+
+	goerrors "github.com/go-errors/errors"
+)
+
+const (
+	newSkip    = 2
+	errorfSkip = 2
+)
+
+// New creates a new instance of Error.
+// If the given value does not contain an stack trace, it will be created.
+func New(val any) error {
+	if val == nil {
+		return nil
+	}
+
+	return newWithSkip(newSkip, val)
+}
+
+// Errorf creates a new error with the given format and values.
+// It can be used as a drop-in replacement for fmt.Errorf() to provide descriptive errors in return values.
+// If none of the given values contains an stack trace, it will be created.
```

Comment:
"a stack trace" (in all places)

Reply:
Ah, thanks, I will fix it in the next PR.